go.mod

```diff
@@ -207,6 +207,7 @@ require (
 	github.com/gorilla/websocket v1.5.0
 	github.com/icholy/digest v0.1.22
 	github.com/meraki/dashboard-api-go/v3 v3.0.9
+	github.com/mjwolf/go-quark v0.2.0
```

Comment:
This will be changed to `github.com/elastic/go-quark` before this is merged. We're still working to finalize publishing Go modules from that repo

Reply:
Thanks, I've switch to use replace, until the `elastic/go-quark` is published

Reply:
`github.com/elastic/go-quark` is now published, and I've switched to use it